content/_includes/site/hire-us.njk

```diff
@@ -0,0 +1,17 @@
+{% import 'embed.macros.njk' as embed %}
+{% import 'layout.macros.njk' as layout %}
+
+{%- set title = '<span>Hire Us to Build</span> Custom Web Apps <span>That Stand the Test of Time</span>' -%}
+{%- set contents -%}
+  - **Well-tested code** = lower maintenance costs
+  - **WCAG accessibility** = more diverse visitors
+  - **Resilient design systems** = better team workflow
+  - **Enhanced performance** = happier customers
+{%- endset -%}
+
+<aside data-layout="cta" aria-labelledby="cta-title" class="hire-us-cta" data-ccs-colors="dark">
+    <h2 id="cta-title">{{ title | mdInline | safe }}</h2>
+    {{ contents | md | safe }}
+
+  {{ layout.section_link("Let's chat", '/contact/') }}
```

Comment:
I'm not sure this link has good contrast against the image. We could add a background to the link, or... Not sure if @SondraE has ideas.

Reply:
I changed this to look like a button  and made a new pattern for that if we think we'll have more link-buttons like this.